src/nodes/display/AnaglyphPassNode.js

```diff
@@ -1,34 +1,19 @@
 import { Fn, nodeObject, vec4 } from '../shadernode/ShaderNode.js';
-import PassNode from './PassNode.js';
-import { Vector2 } from '../../math/Vector2.js';
-import { StereoCamera } from '../../cameras/StereoCamera.js';
-import { HalfFloatType, LinearFilter, NearestFilter } from '../../constants.js';
-import { RenderTarget } from '../../core/RenderTarget.js';
 import { Matrix3 } from '../../math/Matrix3.js';
 import { uniform } from '../core/UniformNode.js';
-import QuadMesh from '../../renderers/common/QuadMesh.js';
 import { uv } from '../accessors/UVNode.js';
 import { clamp, max } from '../math/MathNode.js';
-import { texture } from '../accessors/TextureNode.js';
+import StereoCompositePassNode from './StereoCompositePassNode.js';
+import { addNodeClass } from '../core/Node.js';
 
-const _size = /*@__PURE__*/ new Vector2();
-const _quadMesh = /*@__PURE__*/ new QuadMesh();
+class AnaglyphPassNode extends StereoCompositePassNode {
 
-class AnaglyphPassNode extends PassNode {
+	constructor( scene, camera, eyeSep ) {
```

Comment:
I would favor to remove `eyeSep` from the signature. The instance of `StereoCamera` is public in all three stereo passes and developers can directly set the `eyeSep` via this reference.